### PR TITLE
Bids 1961/time column tooltip consistency

### PR DESCRIPTION
**Pull request by Eisei24**
Please see the comments of the BIDS for a summary of all the relevant pages.

It is of course possible that I missed some page or specific view, maybe you can think of others that view a time.
I could not find any other than the index and ethClients pages that automatically update their time every second but it should be checked that that part also still works for everything.

I could not test the /user/settings#app page which requires paired devices.

### CODE REVIEW

**Review by LuccaBitfly — COMMENTED** [review]
All in all very good PR, could not find anything missing. Please take a look at my comments.

File: static/js/layout.js [hunk 1]
```diff
@@ -510,23 +510,30 @@ function formatTimestamps(selStr) {
...
   })
 
   if (sel.find('[data-toggle="tooltip"]').tooltip) {
     sel.find('[data-toggle="tooltip"]').tooltip()
   }
 }
 
+function formatTimestampsTooltip(local) {
+  var toolTipFormat = "yyyy-MM-dd HH:mm:ss"
+  var tooltip = local.toFormat(toolTipFormat)
```
[the remark this answers is not in the document]
Comment (Eisei24) [reply static/js/layout.js:526]: Yes, I also added a comment to the BIDS.

File: ethClients/ethClients.go [hunk 2]
```diff
@@ -177,21 +176,21 @@ func prepareEthClientData(repo string, name string, curTime time.Time) (string,
...
-		if timeDiff > 30 {
-			return client.Name, fmt.Sprintf("On %s", ymdTodmy(date[0]))
+			timeText = "Recently"
+		} else if timeDiff <= 1.5 && timeDiff >= 1.0 {
+			timeText = "1 day ago"
+		} else if timeDiff > 30 {
+			timeText = fmt.Sprintf("On %s", ymdTodmy(date[0]))
+		} else {
+			timeText = fmt.Sprintf("%.0f days ago", timeDiff) // can sub. -0.5 to round down the days but github is rounding up
 		}
```
[the remark this answers is not in the document]
Comment (Eisei24) [reply ethClients/ethClients.go]: This was discussed and yes, we want to remove the special handling.

**Review by LuccaBitfly — APPROVED** [review]
lgtm!

**Pull request merged**